src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -251,18 +252,16 @@
 
 (defn render-fn
   [{:keys [type value content-type] :as message-data} _ _
-   {:keys [context keyboard-shown? insets]}]
+   {:keys [context keyboard-shown?]}]
   ;;TODO temporary hide mutual-state-updates https://github.com/status-im/status-mobile/issues/16254
   (when (not= content-type constants/content-type-system-mutual-state-update)
-    (if (= type :header)
-      [list-header insets]
-      [rn/view
-       (add-inverted-y-android {:background-color (colors/theme-colors colors/white colors/neutral-95)})
-       (if (= type :datemark)
-         [quo/divider-date value]
-         (if (= content-type constants/content-type-gap)
-           [message.gap/gap message-data]
-           [message/message message-data context keyboard-shown?]))])))
+    [rn/view
+     (add-inverted-y-android {:background-color (colors/theme-colors colors/white colors/neutral-95)})
+     (if (= type :datemark)
```

Comment:
I see not related to this pr, but while you're here can we use `cond` here to avoid nested if's?